test/com/google/enterprise/adaptor/database/UniqueKeyTest.java

```diff
@@ -317,15 +319,15 @@ public void testPreparingRetrieval_wrongColumnCount() throws SQLException {
   @Test
   public void testPreparingRetrieval() throws SQLException {
     executeUpdate("create table data("
-        + "numnum int, strstr varchar, timestamp timestamp, date date, "
-        + "time time, long bigint)");
+        + "numnum int, strstr varchar(20), tymestamp timestamp(3), date date, "
```

Comment:
Let's try to come up with a consistent scheme, rather than be random. This is another place we were lazy and non-standard. SQL keywords are reserved. I don't like adding "col" to each (or many) names. Curious that "timestamp" is an error but "time" is not. I'm not entirely opposed to making the first vowel "y", silly though it is, or adding a prefix or suffix character (0 or z or c?). I would stick to alphanumerics.